Lexplorer/Components/NFTContent.razor

```diff
@@ -35,6 +35,10 @@ else if (hasAnimationContent("model"))
     <model-viewer bounds="tight" enable-pan autoplay src="@nftMetadata?.animationURL" ar ar-modes="webxr scene-viewer quick-look"
                   camera-controls environment-image="neutral" poster="@nftMetadata?.imageURL" shadow-intensity="1"/>
 }
+else if(hasAnimationContent("html"))
+{
+    <iframe allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" frameborder="0" height="100%" sandbox="allow-scripts" src="@nftMetadata?.animationURL" width="100%" style="min-height: 512px;"></iframe>
```

Comment:
Instead of a hardcoded min-height, it would be better to use `@ToPx(height)` like in the "model" case

Reply:
ah missed that.  fbe8e1b has the fix